### ISSUE TITLE
make "there is a problem in the … configuration" user-friendlier

### ISSUE THREAD

**Issue by olets** [olets/git-replay#7]
Currently an excerpt of the yq query is given, for example

```yaml
rebase:
  base:
    - feature # does not exist
```

```
There is a problem in the 'rebase."base".*' configuration.
````

Better would be

```
There is a problem in the 'rebase.base' configuration.
```

**Comment by olets** [comment]
Went with e.g. `'rebase.base.*`. That's clearer when the problem is at the top level: `'rebase.*'` is more helpful than `'rebase'`

**Comment by olets** [comment]
Closed by [v3.1](https://github.com/olets/git-replay/releases/tag/v3.1)
